packages/smooth_app/lib/themes/smooth_theme.dart

```diff
@@ -83,11 +83,10 @@ class SmoothThemes {
           fontWeight: FontWeight.bold,
           color: isDarkTheme ? Colors.white : Colors.black,
         ),
-        bodyText1: TextStyle(
-          color: isDarkTheme ? Colors.white : Colors.black,
-        ),
         bodyText2: TextStyle(
-          color: isDarkTheme ? basicDark : Colors.black,
+          color: isDarkTheme ? Colors.white : Colors.black,
+          fontSize: 14,
+          letterSpacing: 0.5,
```

Comment:
This is just the basic style parameters from bodyText1 applied. Only the font size is a bit smaller, 16 doesn't look good